src/screens/Hooks/OMPSwitchHooks.res

```diff
@@ -1,11 +1,11 @@
 type userInfo = {
   getUserInfo: unit => promise<UserInfoTypes.userInfo>,
-  updateTransactionEntity: unit => unit,
+  updateTransactionEntity: UserInfoTypes.entity => unit,
 }
 let useUserInfo = () => {
   open LogicUtils
   let fetchApi = AuthHooks.useApiFetcher()
-  let {setUserInfoData} = React.useContext(UserInfoProvider.defaultContext)
+  let {setUserInfoData, userInfo} = React.useContext(UserInfoProvider.defaultContext)
   let url = `${Window.env.apiBaseUrl}/user`
```

Comment:
can we add this url in api utils

Reply:
It can't be added in api utils as its creating circular dependencies